### PR TITLE
Summer school: yaml to json and deleted md

### PR DESCRIPTION
**Pull request by korgan00**

### CODE REVIEW

File: components/events/FaqSection.vue [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+
+@Component
+export default class FaqSection extends Vue {
+  khanacademy = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:221]: We should type this with `GeneralLink` from `constants/appLinks.ts`.
```suggestion
  khanacademy: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro',
+    segment: { action: 'events-summer-school > faq > khanacademy-matrix-multiplication-intro' }
+  }
+
+  pythonAndJupyter = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:226]:
```suggestion
  pythonAndJupyter: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html',
+    segment: { action: 'events-summer-school > faq > textbook-python-and-jupyter-notebooks' }
+  }
+
+  linearAlgebra = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:231]:
```suggestion
  linearAlgebra: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://qiskit.org/textbook/ch-prerequisites/linear_algebra.html',
+    segment: { action: 'events-summer-school > faq > qiskit-linear-algebra' }
+  }
+
+  qCEducators = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:236]:
```suggestion
  qCEducators: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://quantum-computing.ibm.com/programs/educators',
+    segment: { action: 'events-summer-school > faq > ibm-quantum-educators-program' }
+  }
+
+  mailtoQCEducators = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:241]:
```suggestion
  mailtoQCEducators: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'mailto:quantum.education@us.ibm.com',
+    segment: { action: 'events-summer-school > faq > mailto-quantum-educators-program' }
+  }
+
+  twitterQiskit = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:246]:
```suggestion
  twitterQiskit: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://twitter.com/qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-twitter' }
+  }
+
+  twitterIBMResearch = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:251]:
```suggestion
  twitterIBMResearch: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 8]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://twitter.com/IBMResearch',
+    segment: { action: 'events-summer-school > faq > ibmresearch-twitter' }
+  }
+
+  githubQiskit = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:256]:
```suggestion
  githubQiskit: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://github.com/Qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-github' }
+  }
+
+  mediumQiskit = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:261]:
```suggestion
  mediumQiskit: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 10]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://medium.com/qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-medium' }
+  }
+
+  qiskitSubscribe = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:266]:
```suggestion
  qiskitSubscribe: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 11]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://qisk.it/sub',
+    segment: { action: 'events-summer-school > faq > qiskit-youtube-subscribe' }
+  }
+
+  joinqiskitslack = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:271]:
```suggestion
  joinqiskitslack: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 12]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://ibm.co/joinqiskitslack',
+    segment: { action: 'events-summer-school > faq > qiskit-slack-join' }
+  }
+
+  qiskitEvents = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:276]:
```suggestion
  qiskitEvents: GeneralLink = {
```

File: components/events/FaqSection.vue [hunk 13]
```diff
@@ -0,0 +1,315 @@
...
+    url: 'https://qiskit.org/events',
+    segment: { action: 'events-summer-school > faq > qiskit-org-events' }
+  }
+
+  mailtoQiskit = {
```
Comment (eddybrando) [inline components/events/FaqSection.vue:281]:
```suggestion
  mailtoQiskit: GeneralLink = {
```

File: pages/events/summer-school.vue [hunk 14]
```diff
@@ -133,62 +118,12 @@ interface AgendaSlot {
...
-    return {
-      eventData,
-      faqData
-    }
   }
 })
 export default class SummerSchoolPage extends QiskitPage {
   routeName = 'summer-school'
   agendaColumnsDataTable: string[] = ['Day', 'Topic', 'Speaker', 'Format']
+  eventData = EventContent
```
Comment (eddybrando) [inline pages/events/summer-school.vue:126]: I believe we should type this.

**Review by eddybrando — CHANGES_REQUESTED** [review]
Thanks for the PR 🙂 

Please take a look at my comments.

Besides that, I have my reservations about using the `constants` directory to store single use data. I personally believe that it should only have constants that may be reuse throughout the codebase by more than 1 consumer.

Otherwise, this constants / variables should live somewhere near the only consumer (the component that imports it).

But at the same time, I'm not that up to date with the strategies discussed for the implementation of this page, so what I'm saying may be contradicting other prior discussions (sorry in advance 😬 ).

Other than that, I'm OK with this PR, but I still miss the big picture, which I will probably get once we review the PR against `main`.

**Comment by korgan00** [comment]
> Thanks for the PR 🙂
> 
> Please take a look at my comments.
> 
> Besides that, I have my reservations about using the `constants` directory to store single use data. I personally believe that it should only have constants that may be reused throughout the codebase by more than 1 consumer.
> 
> Otherwise, this constants / variables should live somewhere near the only consumer (the component that imports it).
> 
> But at the same time, I'm not that up to date with the strategies discussed for the implementation of this page, so what I'm saying may be contradicting other prior discussions (sorry in advance 😬 ).
> 
> Other than that, I'm OK with this PR, but I still miss the big picture, which I will probably get once we review the PR against `main`.

Per offline conversations with @eddybrando , we will keep the file in `constants` folder.

**Pull request merged**